recipes/graphics/xorg/font/font-util.yaml

```diff
@@ -0,0 +1,45 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+   LICENSE: "MIT + BSD-2-Clause"
+   PKG: "font-util"
+   PKG_VERSION: "1.4.1"
+   PD: "Font utilities dirs"
+   COPYRIGHT: "Copyright (c) 2009, Oracle and/or its affiliates.\n
+               Copyright (c) 2003 The NetBSD Foundation, Inc.\n
+               Copyright (c) 2006 Martin Husemann.\n
+               Copyright (c) 2007 Joerg Sonnenberger.\n
+               Copyright 2005 Red Hat, Inc\n
+               Copyright (c) 1991-2011 Unicode, Inc.  All Rights reserved"
+   LICENSE_FILE: "COPYING"
```

Comment:
I guess we need to coordinate these meta environment fields. Could you give some rationale for each of the fields (except `PKG_VERSION` of course)? A couple of thoughts:

* shouldn't all variables be prefixed with `PKG_` consistently?
* Is `LICENSE` supposed to be SPDX compatible?
* The `PKG` looks kind of redundant.
* `PD` == package description? How is that used?

Reply:
>     * shouldn't all variables be prefixed with `PKG_` consistently?
> 

Sounds good. 

>     * Is `LICENSE` supposed to be SPDX compatible?
> 
Yes.

>     * The `PKG` looks kind of redundant.
> 
Not sure if we still need this.

>     * `PD` == package description? How is that used?

It's used to build a list of OpenSourceSoftware used in our products. I think it's also included in the SBOM (SoftwareBillOfMaterials) but I'm not 100% sure. For the [CPE-Field](https://en.wikipedia.org/wiki/Common_Platform_Enumeration) of the SBOM we also use `CPE_VENDOR` and `CPE_PRODUCT` if the recipe name is different to generate the correct cpe-string.

The `COPYRIGHT` field is just what `grep -i copyright` found.. Not sure if this should be included in the upstream recipes.

What do you think about the following for upstream recipes?

   | Variable        | Description                           |    
  |-----------------|---------------------------------------|    
  | PKG_LICENSE     | License of the Package SPDX compatible |    
  | PKG_VERSION     | Package Version                       |    
  | PKG_DESCRIPTION | Short description of the package      |    
  | PKG_LICENSE_FILE| Points to a LICENSE file in the src   workspace.                            |    
  | CPE_VENDOR      | Vendor field of CPE schema            |    
  | CPE_PRODUCT     | Product field of CPE schema           |

Reply:
Sounds all reasonable. I'm not fully sure about the `CPE_` things, though. But these can be added later anyway.

Regarding `PKG_LICENSE_FILE`. I could imagine there is more than one license file in some packages. How should we handle those?

Reply:
True, this is often used if tools have a different license than the lib. So we could add a different license_file to each multipackage?